### ISSUE TITLE
Angular 11 Strict: can't access OverlayScrollbars Inputs

### ISSUE THREAD

**Issue by axelstudios** [KingSora/OverlayScrollbars#281]
**Describe the bug**
If you create a new Angular 11 app with strict type checking (`ng new <name> --strict`) it sets `angularCompilerOptions.strictInputAccessModifiers` to `true` in tsconfig.json.  `strictInputAccessModifiers` honors @Input() access modifiers, and because they are currently set to `private` they are inaccessible:

https://github.com/KingSora/OverlayScrollbars/blob/914836370aa444b04cc9e0c2bca63915ac8fdba0/packages/overlayscrollbars-ngx/src/overlayscrollbars.component.ts#L31-L32

This is the compiler error you'll see:
```html
error TS2341: Property '_options' is private and only accessible within class 'OverlayScrollbarsComponent'.

     <overlay-scrollbars [options]="scrollOptions">
                         ~~~~~~~~~~~~~~~~~~~~~~~~~
```

**To Reproduce**
1. Set `strictInputAccessModifiers` to true in `tsconfig.json` for an Angular 11 app
2. Attempt to use the `options` or `extensions` OS inputs

**Expected behavior**
There should be no template errors when using strict type checking

**Comment by KingSora** [comment]
Good day! 

So If I understand correctly the `private` modifier has to be removed?

**Comment by axelstudios** [comment]
Exactly, `options` and `extensions` have to be public, so removing _private_ would do it

**Comment by KingSora** [comment]
@axelstudios alright... but I've set them to `private` on purpose, because if you get the `ref` to the component, you're in theory able to see and set them. Can I prevent this, or is this valid?

**Comment by axelstudios** [comment]
I've done something similar, I think you would want to keep the variable private and change the Inputs to setters (and no getters).  Something like this:

```typescript
private _options: OverlayScrollbars.Options;

@Input() set options(options: OverlayScrollbars.Options | undefined) {
  if (options) {
    this._options = options;
  }
}
```

**Comment by KingSora** [comment]
Fixed with [version 1.13.1](https://github.com/KingSora/OverlayScrollbars/releases/tag/1.13.1) / `"overlayscrollbars-ngx": "0.2.3"`
